.github/workflows/deploy-package.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow will build pyafipws package and deploy on pypi. 
+# For more information see: https://github.com/py2exe/py2exe/blob/master/.github/workflows/CI.yml
+
+name: Deploy to PyPi
+
+on:
+  workflow_run:
+    workflows: ["python-package", "windows-installer"]
+    types:
+      - completed
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.9, 3.11]
```

Comment:
Probably it should be a single python version (to not publish the package twice)